tests/lib/assertions.py

```diff
@@ -128,6 +128,13 @@ def assert_get_message_response_channels(resp, channelType, channelStatus):
 
     @staticmethod
     def assert_201_response_messages(resp, environment):
+        if "internal-qa" in environment:
```

Comment:
I think this can be simplified if you change line 163 to:
`assert response.get("links").get("self").startswith(environment)`
You can then remove these new lines and lines 154-161.
It looks like I missed `assert_201_response_messages` when altering `assert_200_response_message`.